src/foam/lib/PermissionPredicate.js

```diff
@@ -0,0 +1,55 @@
+/**
+@license
+Copyright 2020 The FOAM Authors. All Rights Reserved.
+http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+foam.CLASS({
+  package: 'foam.lib',
+  name: 'PermissionPredicate',
+  extends: 'foam.mlang.predicate.AbstractPredicate',
+  implements: ['foam.core.Serializable'],
+  imports: [
+    'auth',
+  ],
+  properties: [
+    {
+      name: 'args',
+      class: 'StringArray'
```

Comment:
Better to have only one permission and then AND or OR them together if that's what you need. Otherwise, it isn't clear what it means when you have multiple permissions.

Reply:
fixed https://github.com/foam-framework/foam2/pull/3234